src/styles/block/clock.scss

```diff
@@ -0,0 +1,54 @@
+@mixin handSize {
+  position: absolute;
+  bottom: 50%;
+  left: 40%;
+  border-radius: 10px;
+  width: 5px;
+}
+
+.clock {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+
+  &__center {
+    background-color: black;
+    height: 20px;
+    width: 20px;
+    border-radius: 50%;
```

Comment:
here it would be best if you used z-index to cover the arrows

Reply:
add `position: relative;` and `z-index: 1`. Arrows should be under the clock`s center